### PR TITLE
Python wrapper

### PR DESCRIPTION
**Pull request by filipek92**
Finished enhancement of #108. Thanks @joernesdohr for his wrapper. I've only made examples to work same as original C++ examples.

### CODE REVIEW

**Comment by filipek92** [comment]
Is it possible to merge it into master? @Avamander  If I can do some changes let me know....

**Comment by Avamander** [comment]
I'd gladly merge it, but I have no proper way to test it.

File: examples_RPi/RF24Mesh_Example_Master.py [hunk 1]
```diff
@@ -20,5 +22,8 @@
     mesh.DHCP()
 
     while network.available():
-        print("Received message")
         header, payload = network.read(10)
+        if chr(header.type) == 'M':
+            print("Rcv {} from 0{:o}".format(unpack("L",payload)[0], header.from_node))
```
Comment (KyaTech) [inline examples_RPi/RF24Mesh_Example_Master.py]: At my environment this doesn't work cause the unpack methods requires a string of 4 letters. So to fix that you just need:
`print("Rcv {} from 0{:o}".format(unpack(('L'),payload[:calcsize("L")])[0], header.from_node))`
For this you also need to include this method from the struct-package -> 
`from struct import unpack,calcsize`

Comment (filipek92) [reply examples_RPi/RF24Mesh_Example_Master.py]: How big is your payload? Slave example sends uint32_t, which is 4bytes long?

Which version of python you are using? I'm using only Python 3. Python 2 is bit mysterious for me :-D

Thanks for revision, I'm ready to push a correction

Comment (KyaTech) [reply examples_RPi/RF24Mesh_Example_Master.py]: Well I think the problem could be the python 3 / 2 ... But when I start this script with python3 it doesn't even start (It has problem with finding the imports and with running the modules ...) .  So when I run in python2 this is the only mistake.

Comment (filipek92) [reply examples_RPi/RF24Mesh_Example_Master.py]: Ok, I will try your suggested correction in my instalation as soon as possible, and then push it here.

Thank you

**Pull request merged**